.github/workflows/ci.yml

```diff
@@ -125,25 +127,25 @@ jobs:
             - it-backend: s3
               cassandra-version: 'github:apache/trunk'
             - it-backend: s3
-              python-version: 3.9
+              python-version: "3.10"
```

Comment:
I've changed these excludes from 3.9 to 3.10 now that 3.9 integration tests are no longer being run. Let me know if this was wrong, and I'll revert this part.

Reply:
That's exactly what I had in mind 👍